packages/plugin-ext/src/main/browser/tasks-main.ts

```diff
@@ -175,16 +175,38 @@ export class TasksMainImpl implements TasksMain, Disposable {
     }
 
     protected toTaskConfiguration(taskDto: TaskDto): TaskConfiguration {
-        return Object.assign(taskDto, {
-            _source: taskDto.source,
-            _scope: taskDto.scope
+        const { group, ...taskConfiguration } = taskDto;
+        if (group) {
+            if (taskDto.isBackground === true) {
+                taskConfiguration.group = { kind: group, isDefault: true };
+            } else {
+                taskConfiguration.group = group;
+            }
+        }
+
+        return Object.assign(taskConfiguration, {
+            _source: taskConfiguration.source,
+            _scope: taskConfiguration.scope
         });
     }
 
     protected fromTaskConfiguration(task: TaskConfiguration): TaskDto {
-        return Object.assign(task, {
-            source: task._source,
-            scope: task._scope
+        const { group, ...taskDto } = task;
+        if (group) {
+            if (TaskCustomization.isBuildTask(task)) {
+                taskDto.group = 'build'
```

Comment:
The following is failing the build:

```suggestion
                taskDto.group = 'build';
```

Reply:
Done on new commit !